generate_export_options.rb

```diff
@@ -87,16 +92,23 @@ def export_method(mobileprovision_content)
 fail_with_message('archive_path not specified') unless options[:archive_path]
 puts "archive_path: #{options[:archive_path]}"
 
+fail_with_message('target_os not specified') unless options[:target_os]
+puts "target_os: #{options[:target_os]}"
+
 puts
-puts "\e[34mCollect infos from mobileprovision\e[0m"
+puts "\e[34mCollect provision info\e[0m"
 
-mobileprovision_content = collect_provision_info(options[:archive_path])
+mobileprovision_content = collect_provision_info(options[:archive_path], options[:target_os])
 # team_id = mobileprovision_content['TeamIdentifier'].first
 method = export_method(mobileprovision_content)
 
 
 export_options = {}
 export_options[:method] = method unless method.nil?
+
+# explicitly set this option to false for Xamarin.Mac projects since they have no support of dSYMs
+export_options[:uploadSymbols] = 'NO' if options[:target_os].eql?(Api::MAC)
```

Comment:
yep, logging is quite important - we want to keep our support guys (ahem, us :P) sane ;)